frontend-public/src/components/dashboard/mine/reports/Reports.js

```diff
@@ -90,30 +87,16 @@ export class MineInfo extends Component {
       <div className="mine-info-padding">
         {this.props.mine && (
           <div>
-            <Row>
-              <Col xs={1} sm={1} md={2} lg={4} />
-              <Col xs={22} sm={22} md={14} lg={12}>
-                <h1 className="mine-title">{this.props.mine.mine_name}</h1>
-                <p>Mine No. {this.props.mine.mine_no}</p>
-              </Col>
-              <Col xs={22} sm={22} md={6} lg={4}>
-                <QuestionSidebar />
-              </Col>
-              <Col xs={1} sm={1} md={2} lg={4} />
-            </Row>
-            <Row>
-              <Col xs={1} sm={1} md={2} lg={4} />
-              <Col xs={22} sm={22} md={20} lg={16}>
-                <h2>2018 Reports</h2>
-                <br />
-                <MineInfoList
-                  mine={this.props.mine}
-                  openEditReportModal={this.openEditReportModal}
-                  handleEditReportSubmit={this.handleEditReportSubmit}
-                />
-              </Col>
-              <Col xs={1} sm={1} md={2} lg={4} />
-            </Row>
+            <h1 className="mine-title">{this.props.mine.mine_name}</h1>
+            <p>Mine No. {this.props.mine.mine_no}</p>
+
+            <h2>2018 Reports</h2>
+            <br />
```

Comment:
Is this line break definitely needed? Just looking at the code changes, it looks like an `<h2>` and the wrapping `<div>` for `MineReportTable` should naturally break onto separate lines

Reply:
removed.